extensions/amp-selector/1.0/base-element.js

```diff
@@ -96,7 +96,7 @@ function getOptions(element, mu) {
     .filter(
       (el) =>
         !closestAncestorElementBySelector(
-          pureDevAssert(
+          devAssert(
             el.parentElement?.nodeType == 1 && el.parentElement,
```

Comment:
```suggestion
            el.parentElement && el.parentElement.nodeType == 1,
```

Not crucial for this PR: Is it me or is it kind of funny this is in this order?

Reply:
Agreed, very weird! This was introduced by yours truly in #32474 (see [original assertion](https://github.com/ampproject/amphtml/blob/ea8a3a83c44266d771d17cc8e2ed2008268baeda/extensions/amp-selector/1.0/amp-selector.js#L135)) when rewriting an `assertElement`. Nice catch! dropped the `el.parentElement` altogether since `el.parentElement?.nodeType == 1` captures it all.